providers/catalog.go

```diff
@@ -23,7 +23,7 @@ const (
 	Gong                                Provider = "gong"
 	Zoom                                Provider = "zoom"
 	Intercom                            Provider = "intercom"
-  	Capsule                             Provider = "capsule"
+	Capsule                             Provider = "capsule"
```

Comment:
When I save this file, there is a white space being removed. That is the reason for showing a change.